src/main/java/blackjack/domain/entry/Player.java

```diff
@@ -1,35 +1,35 @@
 package blackjack.domain.entry;
 
 import blackjack.domain.PlayerOutcome;
-import blackjack.domain.card.Card;
 import blackjack.domain.card.HoldCards;
 
-import java.util.List;
-
 public class Player extends Participant {
     private static final int BLACKJACK_NUMBER = 21;
 
     private final String name;
```

Comment:
> 객체지향 생활 체조 원칙 규칙 3: 모든 원시값과 문자열을 포장한다.

위 내용을 적용해볼까요? 그리고 원시값을 포장했을 때의 장점은 무엇일까요?

```suggestion
    private final Name name;
```

코드 전반적으로 앞서 적용했던 객체지향 생활 체조 원칙이 잘 반영되어 있는지 다시 한 번 점검해보세요.

Reply:
원시값을 포장하게 되면 플레이어가 가져야할 이름에 대한 책임이 줄어들게 되고 Name이 스스로 관리를 하게 됩니다!
https://tecoble.techcourse.co.kr/post/2020-05-29-wrap-primitive-type/
자료형에 구애받지 않는 장점도 생기네요!


자동차, 로또 미션에서는 신경 쓴 부분인데 블랙잭에선 규칙3을 많이 신경쓰지 못 했네요. 전체적으로 확인해보겠습니다😭